### DISCUSSION TITLE
My official student id was rejected by Github Education Why

### DISCUSSION THREAD

**Discussion by mr-ben-jeckson** [community/community#77910] [GitHub Education]
### Hi everyone,

GitHub Education rejected my official student ID. I don't understand. I will go to work on my project with AWS Codewhispper and Gitlab. It would be painful if they asked to verify my student ID and cancel the affiliate program. Thanks for everything. I will miss you copilot

**Comment by IgnacioPalma** [comment]
Hello @mr-ben-jeckson! :wave:

Did your verification documentation clearly display your **name**, the **academic institution** you are part of, and a **date**? :thinking:
